index.js

```diff
@@ -379,6 +380,9 @@ module.exports = class NoiseSecretStream extends Duplex {
     const id = buf.subarray(3, 3 + 32)
     streamId(handshakeHash, this.isInitiator, id)
 
+    // initialize secretbox state for unordered messages
+    this._setupSecretSend(handshakeHash)
```

Comment:
self-nitpick, this function doesn't technically have to be a class member anymore:
```
this._sendState = initializeSend(handshakeHash)
```
but the current version is probably easier on the eyes.